### DISCUSSION TITLE
`sails-mongo` compatability issue

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#106] [General]
Hi @neonexus 

I have a Mongo Server whose version is `6.0.5`. But `sails-mongo` is [not compatible](https://github.com/balderdashy/sails-mongo#compatibility) any more as it's been outdated for a while.

In this case, I am planning to use [`MongoClient`](https://mongodb.github.io/node-mongodb-native/3.6/api/MongoClient.html) from the Node.js MongoDB Driver API directly in the code. I know this means, I have to construct all the queries by myself which is an overhead but to use all the latest features, I believe it would be better.

If I use this library to connect with Mongo instead of  a Sails Model, would it have any negative effect on the Sails application as a whole deviating from ORM? I suppose, it shouldn't. Do you have any suggestions or opinions towards this approach?

Thank you.

**Comment by neonexus** [comment]
Have you tried this yet? Looks promising…

https://github.com/balderdashy/sails/issues/7117#issuecomment-1537118722

At the very end of that thread, he made a repo for it as well.

**Reply by MaheshkumarSundaram** [reply]
Thanks @neonexus for pointing it out. I'll try that.

**Reply by neonexus** [reply]
Did you ever get this to work @MaheshkumarSundaram?

**Reply by MaheshkumarSundaram** [reply]
Hi @neonexus,

It did take your suggestion and I used [`sails-mongo-cloud`](https://github.com/sandeeppc/sails-mongo-cloud) to sort that out. So far, I haven't faced any issues with the package. Thank you

**Reply by neonexus** [reply]
👍
